pom.xml

```diff
@@ -233,6 +233,17 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>com.coveo</groupId>
+        <artifactId>fmt-maven-plugin</artifactId>
```

Comment:
Missed this: You don't need to include this plugin into pom.xml.

Reply:
Ben added this in https://github.com/googleapis/java-storage/pull/847/commits/64597019d7e3d69688e3309405221c97fa30806c to fix a `UnsupportedClassVersionError` issue